app/views/tasks/completed.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Here is the big list of things I have to do!!!!</h1>
+<div class="task-container">
+  <%@all_tasks = Task.where.not(completed_at: "").order(completed_at: :desc)%>
```

Comment:
This totally works, but since this is a View it shouldn't interact with the Model at all (as in MVC).  It's hard to get used to at first! But it makes for much better code. In your tasks controller set your variable @all_tasks to interact with the Task model (which I think you did in addition on line 39 in that file!).  Then you can simply use @all_tasks here, and if all_tasks ever changes or is not working, you only have to look in the controller to make changes.